services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+
+from db.models import User
+
+
+@transaction.atomic
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = "",
+                last_name: str = "") -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    try:
+        user = User.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        raise ValueError(f"User with ID {user_id} does not exist.")
```

Comment:
Raising a `ValueError` is appropriate here, but ensure that this is consistent with how you handle similar errors across your application. Consider using custom exceptions for more clarity and consistency.